images/macos/software-report/SoftwareReport.Generator.ps1

```diff
@@ -182,7 +182,8 @@ $toolsList += @(
     (Get-GHCVersion),
     (Get-CabalVersion),
     (Get-StackVersion),
-    (Get-SwiftFormatVersion)
+    (Get-SwiftFormatVersion),
+    (Get-ColimaVersion)

```

Comment:
@bogdan-damian-bgd should it be in the runtime section instead as Colima - container **runtimes** on macOS (and Linux) with minimal setup?

Reply:
Yes, directly from a naming point of view it should be in 'Language and Runtimes'.

On the other hand, if we consider it as some kind of replacement for Docker (which is also labeled as a container **runtime**) and try to be consistent with the location of Docker within the ReadMe sections for Windows and Linux, then it also makes some sense to stay in this section as well?

Reply:
Yup, if we'd like to be consistent then it's fine